blazor/dropdown-list/sorting.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Sorting in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Sorting in Syncfusion Blazor DropDown List component and much more.

```

Comment:
Description: Checkout and learn here all about Sorting in Syncfusion Blazor DropDown List component and much more.